packages/api/package.json

```diff
@@ -25,6 +25,7 @@
     "compression": "^1.7.1",
     "core-js": "^2.5.0",
     "cors": "^2.8.1",
+    "csvtojson": "^1.1.7",
```

Comment:
Moved this to dependencies because it is required by `api/utilities/elasticsearch.js`.